### PR TITLE
[FW-104187] Include user.eids in OpenRTB BidRequest protobuf Message

### PR DESCRIPTION
**Pull request by bsolomon-bw**
See "eids" under User:
https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/b99819955d1a5219301bc05098fb8c2ec6761582/AdCOM%20v1.0%20FINAL.md#object--user-

### CODE REVIEW

File: beeswax/openrtb/openrtb.proto [hunk 1]
```diff
@@ -761,6 +761,18 @@ message BidRequest {
...
+     */
+    optional string genre = 20;
+    /*
+     * Network Object
+     */
+    optional Network network = 21;
+    /*
+     * Channel Object
+     */
+    optional Channel channel = 22;
```
Comment (benlandrum) [inline beeswax/openrtb/openrtb.proto:775]: Are these related?

Comment (bsolomon-bw) [reply beeswax/openrtb/openrtb.proto:775]: Just pulling in the latest version of the pb that we're using internally -- should we not expose these elements?

Comment (benlandrum) [reply beeswax/openrtb/openrtb.proto:775]: That's fine.

**Review by benlandrum — APPROVED** [review]
LGTM.
